### DISCUSSION TITLE
know which address I receive() from

### DISCUSSION THREAD

**Discussion by sto1cLem0n5** [jgromes/RadioLib#227] [Q&A] [answered]
Hi, I'm wondering if it is possible to know the address of the node that sent the message when I call the `receive ( uint8_t * data,
size_t  len )` ? 
Cheers!

**Comment by jgromes** [comment, marked as answer]
At the moment this isn't implemented for any of the modules that support address filtering (SX127x/RFM9x, RF69 and CC1101). The address byte is currently discarded from FIFO upon packet reception.
